### DISCUSSION TITLE
My next/react native web project is broken

### DISCUSSION THREAD

**Discussion by Razorshnegax018** [vercel/next.js#44254] [Help] [answered]
Hello,

I was originally going to name this discussion, 'next with react native web doesn't work' but that is misleading, because it does indeed work.

 I have a [react native web project](https://codesandbox.io/p/github/Razorshnegax018/north-sea-native-next/draft/vigorous-williams) that has been in the works for a long time. Time and time again my vision for the project changes, and so does all my code. I now have a solid goal and roadmap for what I want it to be now, and I thought I could use nextjs to accelerate my path and deliver a better experience. However, right after porting the project, I was beset by errors that threw a wrench in my development.

First is the next/images plugin. As per the expo docs, I have installed the plugin and loaded it in my next config, but I still get errors saying that my image import is invalid. More on that [here](https://github.com/expo/expo-cli/issues/4613) and [here](https://stackoverflow.com/questions/74837692/nextjs-image-import-is-not-a-valid-image-file). But that's not the only problem that's been plaguing my project.

 Whenever I try to even run yarn next dev,  my console is mercilessly and brutally spammed with the same error over and over again *on top of* the image error that prevents my project from building. The webpack error is as following:

`[webpack.cache.PackFileCacheStrategy] Restoring failed for CopyFilePlugin|/project/home/razorshnegax018/workspace/north-sea/next/node_modules/next/dist/build/polyfills/polyfill-nomodule.js from pack: TypeError: Cannot read properties of undefined (reading 'length')`

and 

`[webpack.cache.PackFileCacheStrategy] Caching failed for pack: Error: ENOENT: no such file or directory, rename '/project/home/razorshnegax018/workspace/north-sea/next/.next/cache/webpack/client-development-fallback/0.pack_' -> '/project/home/razorshnegax018/workspace/north-sea/next/.next/cache/webpack/client-development-fallback/0.pack'`

The second error only appears once, while the first one completely gangbangs my console screen.

At this point, there is almost no reason for me to have a bunch of useless boilerplate code for a framework that doesn't even work. I want to take advantage of all that nextjs has to offer like built-in middleware, excellent routing, deployment on vercel, and SSR but I'm not using any of those features right now. I'm just using npm run web.

I've opened and issue in expo/cli and a question on stackoverflow, both linked above and neither have been touched by anyone other than myself. This is my last resort. If you've read this far, I implore you to lend a helping hand Thank you

**Comment by GabenGar** [comment]
Have you tried deleting `node_modules` and `.next` folders first?

**Reply by Razorshnegax018** [reply]
I have. It did fix the webpack errors, but I am unfortunately still unable to build my project because of the image error

**Reply by dadyutenga** [reply]
Did you fix it 😑

**Reply by Razorshnegax018** [reply, marked as answer]
I did. See [here](https://github.com/expo/expo-cli/issues/4613)

**Reply by dadyutenga** [reply]
Going to implement this
